INSTALLATION.md

```diff
@@ -13,6 +13,25 @@ This document provides instructions on how to set up and start a running instanc
   - [Install node.js](#install-nodejs)
   - [Install TypeScript](#install-typescript)
   - [Install Required Packages](#install-required-packages)
+- [Configuration](#configuration)
+  - [Creating .env file](#creating-env-file)
+  - [Setting up PORT in .env file](#setting-up-port-in-env-file)
+  - [Setting up REACT_APP_TALAWA_URL in .env file](#setting-up-react_app_talawa_url-in-env-file)
+  - [Setting up REACT_APP_BACKEND_WEBSOCKET_URL in .env file](#setting-up-react_app_backend_websocket_url-in-env-file)
+  - [Setting up REACT_APP_RECAPTCHA_SITE_KEY in .env file](#setting-up-react_app_recaptcha_site_key-in-env-file)
```

Comment:
Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.